mousestyles/dynamics/__init__.py

```diff
@@ -337,3 +337,92 @@ def mcmc_simulation(mat_list, n_per_int):
             simu_result[index] = int(state_j)
     simu_result = np.array(simu_result, dtype=int)
     return simu_result
+
+
+def get_score(true_day, simulated_day, weight=[1, 10, 50, 1]):
```

Comment:
I will explain that both in the doc string and in the documentations. I choose the initial weights based on the ratio of different status in the mouse day. say like, the average mouse day have 21200 timestamps. 10000 of them are IS, 1000 are EAT, 200 are drink, and the left 10000 are OTHERS. So i think we should weigh more on drink and eat, their ratio is 10000:1000:200:10000 = 1:0.1:0.02:0.1. So I took the inverse of them to be 1:10:50:1.